src/components/Main/MyPresentation/index.jsx

```diff
@@ -56,6 +123,11 @@ function MyPresentation({ presentations }) {
           );
         })}
       </Container>
+      {contextMenu.visible && (
+        <ContextMenu style={{ top: contextMenu.y, left: contextMenu.x }}>
```

Comment:
```suggestion
        <ContextMenu contextMenu={contextMenu}>
```
style 속성을 하단의 styled-components로 넣어주세요!

Reply:
감사합니다! 이부분 생각하지 못했었는데 짚어주셔서 감사해요~ 말씀해주신대로 수정했습니다!